client/common/src/util/markdown/markdown.test.ts

```diff
@@ -69,40 +69,35 @@ describe('renderMarkdown', () => {
             <td>B</td>
             </tr>
             </tbody></table>
-            <p><img src=\\"./src.jpg\\" alt=\\"image alt text\\" /></p>
-            "
+            <p><img alt=\\"image alt text\\" src=\\"./src.jpg\\"></p>"
         `)
     })
     it('renders to plain text with plainText: true', () => {
-        expect(renderMarkdown('A **b**', { plainText: true })).toBe('A b\n')
+        expect(renderMarkdown('A **b**', { plainText: true })).toBe('A b')
     })
     it('sanitizes script tags', () => {
         expect(renderMarkdown('<script>evil();</script>')).toBe('')
     })
     it('sanitizes event handlers', () => {
-        expect(renderMarkdown('<svg><rect onclick="evil()"></rect></svg>')).toBe('<p><svg><rect></rect></svg></p>\n')
+        expect(renderMarkdown('<svg><rect onclick="evil()"></rect></svg>')).toBe('<p></p>')
```

Comment:
suggestion: Match test description to actual test

```suggestion
        expect(renderMarkdown('<b onclick="evil()">test</b></svg>')).toBe('<p><b>test</b></p>')
```